blueprint/src/chapters/chapter1/sheet4.tex

```diff
@@ -0,0 +1,21 @@
+\section{Some Examples of Groups}
+
+\begin{definition}[Integers]
+    \label{definition : Int}
+    \lean{Int}
+    \leanok
+    \uses{definition : Group}
+    $(\mathbb{Z}, +)$, the integers with addition, is a group. $0$ is the identity,
+    and the inverse of x is -x.
+\end{definition}
+
+\begin{definition}[$C_3$]
+    \label{definition : C3}
+    \lean{AlgebraInLean.C3}
+    \leanok
+    \uses{definition : Cn}
+    $C_3$ is the group of rotational symmetries of an equilateral triangle under
```

Comment:
rather than saying "under" composition, you should be more verbose here and say, "with the group operation being the composition of symmetries"